### DISCUSSION TITLE
Release v3.11.2

### DISCUSSION THREAD

**Discussion by harazdovskiy** [serverless-operations/serverless-step-functions#549] [General]
Hi guys!
I'm wondering when we can expect `master` to be released as a `v3.11.2` ? I'm really waiting for the new Distributed map to be available! Some changes happened [during the last 2 months](https://github.com/serverless-operations/serverless-step-functions/compare/v3.11.1...master) and it really worth checking out!
Also asl-validator team [added support](https://github.com/ChristopheBougere/asl-validator/issues/115) to that one. So that definition would be validated smoothly and with no issues as was mentioned [here](https://github.com/serverless-operations/serverless-step-functions/pull/539#issuecomment-1372279672)!

@lopburny, @bahrmichael, @nicksiscoe, @raphaeldeem-acorns

**Comment by lopburny** [comment]
Hi, @harazdovskiy. I have kept you waiting, but I've just released a new version as `v3.12.0` since the new version contains some new features 🙌

**Reply by horike37** [reply]
@lopburny 
Well done!

**Reply by harazdovskiy** [reply]
Guys, thanks! You are awesome! Going to try it today soon!

**Reply by harazdovskiy** [reply]
@lopburny, seems to work fine 🚀
